### DISCUSSION TITLE
Step by step PowerShell code for setting up multiple machines to plot

### DISCUSSION THREAD

**Discussion by CryptoKnightMining** [Chia-Network/chia-blockchain#1681] [Q&A]
I have limited coding experience and have been successfully Plotting and Farming on a "main machine" but I would like to set up multiple "Harvester" machines. Can anyone with experience coding this with command line or PowerShell help me by providing step by step code that I can use to make this happen?

**Comment by hoffmang9** [comment]
This is the best we have right now - https://github.com/Chia-Network/chia-blockchain/wiki/Farming-on-many-machines

This process is way more painful and incomplete than we want it but things like transactions and pools are ahead of it.

**Reply by CryptoKnightMining** [reply]
Thank you!
